### PR TITLE
[KV] Add Data Security Page

### PR DESCRIPTION
**Pull request by markdembo**
* Add data security page
* Remove security section from "How KV works"

### CODE REVIEW

File: content/kv/reference/how-kv-works.md [hunk 1]
```diff
@@ -59,6 +59,3 @@ If you need stronger consistency guarantees, consider using [Durable Objects](/d
 
 An approach to achieve write-after-write consistency is to send all of your writes for a given KV key through a corresponding instance of a Durable Object, and then read that value from KV in other Workers. This is useful if you need more control over writes, but are satisfied with KV's read characteristics described above.
 
-## Security
```
Comment (elithrar) [inline content/kv/reference/how-kv-works.md:62]: This should link over to the new data security page?

**Review by elithrar — APPROVED** [review]

File: content/kv/reference/how-kv-works.md [hunk 2]
```diff
@@ -59,6 +59,3 @@ If you need stronger consistency guarantees, consider using [Durable Objects](/d
 
 An approach to achieve write-after-write consistency is to send all of your writes for a given KV key through a corresponding instance of a Durable Object, and then read that value from KV in other Workers. This is useful if you need more control over writes, but are satisfied with KV's read characteristics described above.
```
Comment (elithrar) [inline content/kv/reference/how-kv-works.md:62]:
```suggestion
## Security

Visit the [data security documentation](/kv/reference/data-security/) to understand how Workers KV secures data.
```

File: content/kv/reference/data-security.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Encryption in Transit
+
+Data transfer between a Cloudflare Worker, and/or between nodes within the Cloudflare network and KV is secured using the same [Transport Layer Security](https://www.cloudflare.com/learning/ssl/transport-layer-security-tls/) (TLS/SSL).
+
+API access via the HTTP API or using the [wrangler](/workers/wrangler/install-and-update/) command-line interface is also over TLS/SSL (HTTPS).
+
+## Compliance
+
+To learn more about Cloudflare's adherence to industry-standard security compliance certifications, visit the Cloudflare [Trust Hub](https://www.cloudflare.com/trust-hub/compliance-resources/).
```
Comment (Maddy-Cloudflare) [inline content/kv/reference/data-security.md]:
```suggestion
To learn more about Cloudflare's adherence to industry-standard security compliance certifications, refer to Cloudflare's [Trust Hub](https://www.cloudflare.com/trust-hub/compliance-resources/).
```

File: content/kv/reference/how-kv-works.md [hunk 4]
```diff
@@ -58,7 +58,6 @@ If you need stronger consistency guarantees, consider using [Durable Objects](/d
 {{</Aside>}}
 
 An approach to achieve write-after-write consistency is to send all of your writes for a given KV key through a corresponding instance of a Durable Object, and then read that value from KV in other Workers. This is useful if you need more control over writes, but are satisfied with KV's read characteristics described above.
-
 ## Security
 
-All values are encrypted at rest with 256-bit AES-GCM, and only decrypted by the process executing your Worker code or responding to your API requests.
\ No newline at end of file
+Visit the [data security documentation](/kv/reference/data-security/) to understand how Workers KV secures data.
```
Comment (Maddy-Cloudflare) [inline content/kv/reference/how-kv-works.md]:
```suggestion
Refer to [Data security documentation](/kv/reference/data-security/) to understand how Workers KV secures data.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**